run.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
-echo "$(dirname "$0")"
-source venv/bin/activate
-python whisper-dictation.py
+Directory=$(dirname $0)
```

Comment:
```suggestion
directory=$(dirname $0)
```

Would be great to use more common formatting for variables